cmd/tea/oapi_compose_test.go

```diff
@@ -56,7 +54,26 @@ func Test_runOAPICompose(t *testing.T) {
 				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+			defer func() {
+				if cleanupAfterTest {
+					require.NoError(t, os.Remove(outputFilePath))
+				}
+			}()
 			assert.NoError(t, tt.cond(t))
+
+			tofp, err := os.ReadFile(tt.args.testOutputFilePath)
+			if err != nil {
+				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+
+			ofp, err := os.ReadFile(tt.args.opts.OutputFilePath)
+			if err != nil {
+				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
```

Comment:
this one could fail, but the message should reflect that (its not an error of `runOAPICompose` directly, but a missing result file)